### PR TITLE
Add recipe files for ffnet

### PR DESCRIPTION
**Pull request by SolarDrew**

### CODE REVIEW

File: recipes/ffnet/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install
```
Comment (synapticarbors) [inline recipes/ffnet/bld.bat]: I think it is reasonable to remove both the `bld.bat` and `build.sh` in favor of just using the `script` entry in the `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

File: recipes/ffnet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: Add `script`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26

File: recipes/ffnet/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+requirements:
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: I think you need `setuptools` and `toolchain` in the `build` section. Also is `networkx` required to build the package or just run it. If it's the latter, please delete it from the `build` list.

File: recipes/ffnet/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  imports:
+    - ffnet
+    - ffnet.examples
+    - ffnet.fortran
+
+about:
+  home: ffnet.sourceforge.net
+  license: GNU Lesser General Public License v3 (LGPLv3)
+  license_family: LGPL
+  license_file: ''
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: `license_file: LICENSE`

File: recipes/ffnet/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - ffnet
+    - ffnet.examples
+    - ffnet.fortran
+
+about:
+  home: ffnet.sourceforge.net
+  license: GNU Lesser General Public License v3 (LGPLv3)
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: `license: LGPL-3.0`

File: recipes/ffnet/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    >>> target  = [ [1.], [0.], [0.], [1.] ]\r\n>>> net.train_tnc(input, target, maxfun = 1000)\r\n>>> net.test(input, target, iprint = 2)\r\n>>> savenet(net, \"xor.net\")\r\n>>> exportnet(net, \"xor.f\"\
+    )\r\n>>> net = loadnet(\"xor.net\")\r\n>>> answer = net( [ 0., 0. ] )\r\n>>> partial_derivatives = net.derivative( [ 0., 0. ] )\r\n\r\nFor instalation instructions and documentation go to `http://ffnet.sourceforge.net\
+    \ <http://ffnet.sourceforge.net>`_.\r\n\r\n**Note: ffnet has now a graphical user interface** `ffnetui <https://pypi.python.org/pypi/ffnetui>`_.\r\n\r\n\r\n\r\nRelease notes\r\n-------------\r\n\r\n\
+    0.8.3\r\n+++++\r\nLicense switched to LGPL. Only minor updates in code are performed, no API changes are made. Along with this version, new graphical user interface ffnetui-0.8.3 has been released.\r\
+    \n\r\n0.8.0\r\n+++++\r\nThis version supports python 3. Only minor changes in code (and no API changes) are made in comparison to previous release, all scripts should run without problems."
+  doc_url: ''
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: Just delete `doc_url` and `dev_url` if there are no links.

File: recipes/ffnet/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+  dev_url: ''
+
+extra:
+  recipe-maintainers: ''
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: You need to add yourself as a recipe maintainer at a minimum:

```
extra:
   recipe-maintainers:
      - SolarDrew
```

**Comment by SolarDrew** [comment]
Thanks for the review @synapticarbors. I'm new to conda packaging so I really appreciate the feedback :D I'lll try and get round to making these changes in the next few days.

File: recipes/ffnet/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,55 @@
+{% set name = "ffnet" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "1aaf35158239cd1e6f63504e884683abac87c77919af36cbefbf1169fed5d1a2" %}
```
Comment (synapticarbors) [inline recipes/ffnet/meta.yaml]: it looks like the `sha256` should be:

4d7befca0247108fc05fb7ebe7cdcff78eb642c4ab8c93215f592dcfed0f29f6

I think you were grabbing the value for the `.zip` copy of the source.

**Pull request closed without merging**